plugins/dell-kestrel/fu-dell-kestrel-plugin.c

```diff
@@ -384,6 +384,11 @@ fu_dell_kestrel_plugin_composite_prepare(FuPlugin *plugin, GPtrArray *devices, G
 			if (!fu_dell_kestrel_ec_run_passive_update(FU_DELL_KESTREL_EC(ec_dev),
 								   error))
 				return FALSE;
+
+			/* send twice to ensure the target is aware */
+			if (!fu_dell_kestrel_ec_run_passive_update(FU_DELL_KESTREL_EC(ec_dev),
+								   error))
+				return FALSE;
```

Comment:
why not just do this directly in `fu_dell_kestrel_ec_run_passive_update`?

Reply:
sounds good, will move.